Formula/f/fd.rb

```diff
@@ -24,7 +24,10 @@ class Fd < Formula
   def install
     system "cargo", "install", *std_cargo_args
 
+    # Generate bash / fish completion using default paths, ZSH comp is included in source files
+    # Also install non-standard powershell completion as fd.ps1 → _fd.ps1
     generate_completions_from_executable(bin/"fd", "--gen-completions", shells: [:bash, :fish])
+    (share/"powershell/completions/_fd.ps1").write Utils.safe_popen_read(bin/"fd", "--gen-completions", "powershell")
```

Comment:
This will work I guess.
```suggestion
    generate_completions_from_executable(bin/"fd", "--gen-completions", shells: [:bash, :fish, :pwsh])
```

Reply:
This will result in a `fd.ps1` completions file being linked, instead of the standard format of `_fd.ps1`, is that okay?

Reply:
Can you give me a reference for that? Then we should fix `brew` itself.

Reply:
I have added a comment to the pull request itself, please let me know which solution we should go forward with for the time being.

I'd recommend using the private API for the time being, and replacing the usage once `generate_completions_from_executable` is fixed, or simply accepting the incorrect filename format for now, while the proposal is being discussed.

If it would be more appropriate to separate the proposal I've included in my comment into an issue, please let me know if this repository is the correct place to open such an issue, or if I should open it elsewhere.

Reply:
No worries, just give me some time.
- https://github.com/Homebrew/brew/pull/20008

Reply:
Thank you! I appreciate you taking the time to look into it :)

I'll update this PR and get it ready for merging once #20008 is resolved

Reply:
Now it works.
```
ls /opt/homebrew/share/pwsh/completions
 _fd.ps1
```